cron/hourly/70-cancel-subscriptions-for-cancelled-orders.ts

```diff
@@ -79,7 +97,7 @@ export async function run() {
             subscription: order.Subscription,
             collective: order.collective.minimal,
             fromCollective: order.fromCollective.minimal,
-            reasonCode: 'ARCHIVED_ACCOUNT',
+            reasonCode: 'ORDER_CANCELLED',
```

Comment:
Would be great to keep a rich value for this one, we want to use it to track statistics on **why** orders get cancelled. An option can be, from `getOrderCancelationReason`, to return a tuple like `[reasonCode, resonMessage]`